static/skin/index.js

```diff
@@ -1,4 +1,24 @@
 (function() {
+    class FragmentParams extends Map {
+        constructor(fragment = '') {
+            if (fragment[0] == '#')
+                fragment = fragment.substring(1); // remove #
+            const nmap = new Map (fragment
+                .split("&")
+                .map(v => v.split("=")));
+            super(nmap);
+        }
```

Comment:
UrlSearchParam properly handle url encoded parameters (see the note in https://url.spec.whatwg.org/#urlsearchparams). We should do the same for the fragments (constructor and tostring).